class LanguageIdentifier in package language is deprecated

```
[WARNING] /home/nruest/git/aut/src/main/scala/io/archivesunleashed/spark/matchbox/DetectLanguage.scala:24: warning: class LanguageIdentifier in package language is deprecated: see corresponding Javadoc for more information.
[INFO]     else new LanguageIdentifier(input).getLanguage
[INFO]              ^
[WARNING] one warning found
```

--- a/pom.xml
+++ b/pom.xml
@@ -499,12 +499,12 @@
     <dependency>
       <groupId>org.apache.tika</groupId>
       <artifactId>tika-core</artifactId>
-      <version>1.16</version>
+      <version>1.12</version>
     </dependency>
     <dependency>
       <groupId>org.apache.tika</groupId>
       <artifactId>tika-parsers</artifactId>
-      <version>1.16</version>
+      <version>1.12</version>
     </dependency>
     <dependency>  <!-- needed for running boilerpipe, but will compile without -->
       <groupId>com.syncthemall</groupId>

